netmaster/master/network.go

```diff
@@ -183,7 +190,8 @@ func CreateNetwork(network intent.ConfigNetwork, stateDriver core.StateDriver, t
 		return nil
 	}
 
-	if GetClusterMode() == "docker" {
+	mode := GetClusterMode()
```

Comment:
Actually I can revert this back. I was initially checking for docker-compose as well. But it is not required now.